### DISCUSSION TITLE
Scale interaction?

### DISCUSSION THREAD

**Discussion by paulmelis** [StereoKit/StereoKit#393] [Q&A]
What would be a good way to add scale interaction to an SK app? `UI.Handle` updates a `Pose`, but that only stores a rotation and translation making me wonder why that is. I could imagine it's because scaling usually requires two-hand interaction (i.e. grab with two hands and move apart), but using two `UI.Handle`s each showing a small handle then becomes a bit tricky, although doable.

**Comment by paulmelis** [comment]
@maluoi Somewhat related to the normal transform discussion 🙂

**Reply by maluoi** [reply]
Sorry, I missed this one earlier!

I think the current situation is perhaps a bit too much of a focus on `Pose` on my part. I initially had this idea that scaling an object wasn't a "real" thing that happens, and so I wanted to discourage it to some extent. But, that kind of thinking probably doesn't belong super well at the SK level! I still don't actually have a full `Transform` object that includes scale at all, and this is something I'm planning on correcting with the next UI update, along with `Handles` that scale.

There's definitely ways to do scale based transforms right now, but only if you're doing it from scratch! You might be able to repurpose a thing or two, but I don't have a good picture of what this would look like off the top of my head. So far I've been using UI sliders for scale.
